changelog.md

```diff
@@ -1,3 +1,18 @@
+9.6.0
+======
+## New features
+
+* Added new authorization header provider interface  `IAuthorizationHeaderProvider2` supporting token return with binding certificate, expanding certificate-based authentication scenarios. For details, see [PR #223](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/pull/223) and [PR #228](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/pull/228).

```

Comment:
@jmprieur Would `IAuthorizationHeaderProvider2` be used for anything or is it obsolete? It's not clear from the notes or interface comments.